kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/utils/scheduling/Task.kt

```diff
@@ -104,9 +106,19 @@ public open class Task(
                 }
             }
 
-            removeFromParent()
+            if (!repeat) {
+                removeFromParent()
+            }
 
             job = null
+
+            if (repeat) {
+                sentryContext.breadcrumb(BreadcrumbType.Info) {
```

Comment:
I think this could result in a memory leak, considering you add a breadcrumb every time the task repeats. I think it'd probably be simplest to just remove it, but I'm open to better ideas.